app/auth-pages/components/register-page.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button, ButtonGroup, Form, Link } from '@servicetitan/design-system';
+import * as React from 'react';
+
+import { enumToOptions } from '../../common/utils/form-helpers';
+import { Role } from '../../user/models';
+
+import { AuthLayout } from './auth-layout';
+
+export const RegisterPage = () => {
+    const register = console.log; // @todo
+
+    return (
+        <AuthLayout title="Register">
+            <Form onSubmit={register}>
+                <Form.Input label="Login" />
+                <Form.Input label="Password" type="password" />
+                <Form.Input label="Password Confirmation" type="password" />
+                <Form.Select
+                    label="Role"
+                    options={enumToOptions(Role)}
```

Comment:
It would be better to make const w/ result of enumToOptions(Role) to avoid this operation on each render.